docs/Concepts/3.md

```diff
@@ -1,46 +1,23 @@
 ---
-title: Content Topic and How to Choose One
+title: Protocols Overview by Example
 ---
 
-A content topic is used for content based filtering and allows you to filter out the messages that your dApp processes,
-both when receiving live messages (Relay or Filter) or retrieving historical messages (Store).
+We have six nodes, A-F.
 
-The recommended format for content topics is as follows:
+The protocols initially mounted are indicated as such.
 
-`/{dapp-name}/{version}/{content-topic-name}/{encoding}`
+![protocol overview](../../static//sequnce-charts/protocols-overview/overview.png "Protocols Overview")
```

Comment:
Please use the mermaid chart feature instead of separate image

Reply:
This included having to update all docuaurus packages to `2.2.0` (latest LTS) as that is where mermaid support comes in